com.getmailspring.Mailspring.yml

```diff
@@ -44,8 +44,6 @@ modules:
       - install -Dm755 start-mailspring /app/bin/start-mailspring
       # Modify appdata and desktop file
       - sed -i 's/<id>mailspring/<id>com.getmailspring.Mailspring/' /app/share/appdata/mailspring.appdata.xml
-      - sed -i 's/<screenshot type="default">/<screenshot type="default"><caption>Mailspring Hero<\/caption><image type="source">/' /app/share/appdata/mailspring.appdata.xml
-      - sed -i 's/<\/screenshot>/<\/image><\/screenshot>/' /app/share/appdata/mailspring.appdata.xml
```

Comment:
use a patch for this in the future https://docs.flatpak.org/en/latest/module-sources.html#patch-sources